AUTHORS.rst

```diff
@@ -11,8 +11,10 @@ Active contributors:
 
 * David Caro <david@dcaro.es>
 * Esteban J. G. Gabancho <esteban.jose.garcia.gabancho@cern.ch>
+* Jacopo Notarstefano <jacopo.notarstefano@gmail.com>
 * Jiri Kuncar <jiri.kuncar@cern.ch>
 * Sami Hiltunen <sami.mikael.hiltunen@cern.ch>
 * Samuele Kaplun <samuele.kaplun@cern.ch>
 * Tibor Simko <tibor.simko@cern.ch>
+* zazasa <salvatore.zaza@gmail.com>
```

Comment:
K. In the past @tiborsimko was pinging developers to confirm this is how they want to be presented in the log. 

I'll ping Salvatore :+1: